engine/src/main/scala/cromwell/engine/db/slick/ExecutionInfoComponent.scala

```diff
@@ -45,6 +47,16 @@ trait ExecutionInfoComponent {
     } yield (execution, executionInfo)
   )
 
+  val runningExecutionsAndExecutionInfosByWorkflowId = Compiled(
+    (id: Rep[String]) => for {
+      executionInfo <- executionInfos
+      execution <- executionInfo.execution
+      workflowExecution <- execution.workflowExecution
+      if workflowExecution.workflowExecutionUuid === id
```

Comment:
I'll check to make sure this is working as you described and I'll add a comment and/or refactor it a bit